Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveE.cpp

```diff
@@ -84,7 +84,7 @@ void FiniteDifferenceSolver::MacroscopicEvolveECartesian (
     // properties from their respective staggering to the Ex, Ey, Ez locations
     amrex::GpuArray<int, 3> const& sigma_stag = macroscopic_properties->sigma_IndexType;
     amrex::GpuArray<int, 3> const& epsilon_stag = macroscopic_properties->epsilon_IndexType;
-    amrex::GpuArray<int, 3> const& mu_stag = macroscopic_properties->mu_IndexType;
+    //amrex::GpuArray<int, 3> const& mu_stag = macroscopic_properties->mu_IndexType;
```

Comment:
Let's remove the line, this is now defined in `Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.cpp` directly.
```suggestion
```